frontier_exploration:
  ros__parameters:
    ~debug: false

    # Rates
    rate: 0.5
    num_explore_fails_before_finish: 5

    # Publishers
    frontier_cells_publiser: "/frontier_exploration/frontier_cells"
    frontier_start_publisher: "/frontier_exploration/start"
    frontier_goal_publisher: "/frontier_exploration/goal"
    cspace_publisher: "/cspace"
    cost_map_publisher: "/cost_map"

    # A* params
    a_star_cost_weight: 10.0
    frontier_size_cost_weight: 1.0
    max_num_frontiers_to_check: 8